relay-general/src/store/normalize/breakdowns.rs

```diff
@@ -202,15 +202,19 @@ impl EmitBreakdowns for SpanOperationsConfig {
 
 /// Configuration to define breakdown to be generated based on properties and breakdown type.
 #[derive(Debug, Clone, Serialize, Deserialize)]
-#[serde(tag = "type", rename_all = "snake_case")]
+#[serde(tag = "type", rename_all = "camelCase")]
 pub enum BreakdownConfig {
+    #[serde(alias = "span_operations")]
     SpanOperations(SpanOperationsConfig),
+    #[serde(other)]
+    Unsupported,
 }
 
 impl EmitBreakdowns for BreakdownConfig {
     fn emit_breakdowns(&self, event: &Event) -> Option<Measurements> {
         match self {
             BreakdownConfig::SpanOperations(config) => config.emit_breakdowns(event),
+            _ => None,
```

Comment:
```suggestion
            BreakdownConfig::Unsupported => None,
```

If new breakdown variant is added, one should not forget to implement it..